data-table/__tests__/table.js

```diff
@@ -1,63 +1,80 @@
-// test 4-5 function
 const table = require('../table.js');
 
 beforeEach(() => {
     document.body.innerHTML = '<div id="div-id"></div>'
 });
 
-describe('test createDataTable', () => {
-      
-  test('test0', () => {
-    const config = {
-        'wrapperDivId': 'div-id',
-    };
-    table.createDataTable(config);
-    const contents = document.getElementById(config.wrapperDivId).textContent;
-    expect(contents.substr(0, 10)).toEqual("Add column");
-  });
+describe('basic tests to ensure createDataTable can function well', () => {
+    test('check the add colunm button', () => {
+        const config = {
+            'wrapperDivId': 'div-id',
+        };
+        table.createDataTable(config);
+        const contents = document.getElementById(config.wrapperDivId).textContent;
+        expect(contents.substr(0, 12)).toEqual("+ Add column");
+    });
+    test('check the default value of rows', () => {
+        const config = {
+            'wrapperDivId': 'div-id',
+        };
+        table.createDataTable(config);
+        const contents = document.getElementsByTagName('table')[0].rows.length;
+        expect(contents).toEqual(4);
+    });
 
-  test('test1', () => {
-      const config = {
-          'wrapperDivId': 'div-id',
-      };
-      table.createDataTable(config);
-      let resConfigDict = table.configDict;
-      let resConfig = resConfigDict[config.wrapperDivId];
-      expect(resConfig.numRows).toEqual(3);
-      expect(resConfig.numColumns).toEqual(4);
-      expect(resConfig.rowsName).toEqual("row");
-      expect(resConfig.columnsName).toEqual("column");
-  });
 
-  test('test2', () => {
-    const config = {
-        'wrapperDivId': 'div-id',
+    test('check the default value of colunms', () => {
+        const config = {
+            'wrapperDivId': 'div-id',
+        };
+        table.createDataTable(config);
+        const contents = document.getElementsByTagName('table')[0].rows[0].cells.length;
+        expect(contents).toEqual(4);
+    });
+
+    test('properly test the first cell', () => {
+        table.createDataTable({
+            'wrapperDivId': 'div-id'
+        });
+        const contents = document.getElementsByClassName('data-table-cell')[0].textContent;
+        expect(contents).toEqual('Rows');
+    });
+    
+    test('properly test the first cell', () => {
+        table.createDataTable({
+            'wrapperDivId': 'div-id'
+        });
+        const contents = document.getElementsByClassName('data-table-cell')[1].textContent;
+        expect(contents).toEqual('Column 1');
+    });
+    
+    test('test that those IDs are actually present', () => {
```

Comment:
Can you clarify what "those IDs" means here? What is the significance of these two divs, and what is this testing that other tests haven't covered?